server/data/data-jmap-cassandra/src/test/resources/json/event.json

```diff
@@ -1,7 +1,7 @@
 {
   "type":"filtering-rule-set-defined",
   "eventId":0,
-  "aggregateId":"FilteringRule/Bart",
+  "aggregateId":"FilteringRule/bart",
```

Comment:
Let say that both cassandra and jmap-draft are "experimental" and thus that users with capital letters loosing their "jmap filters" is oK ^^

Reply:
Should we document it somewhere ?

Reply:
We should document anyway that users with lower case will need MUA reconfiguration on top of JPA-Guice.